.travis.yml

```diff
@@ -5,8 +5,8 @@ language: scala
 sudo: false
 
 scala:
-  - "2.12.11"
-  - "2.13.3"
+  - "2.12.14"
+  - "2.13.6"
```

Comment:
that seems like a strange thing to change in this PR?

Reply:
true, I'll Pr separately. 


Still, the .travis.yml is the only place where these scala versions are used. All other references (projects/Dependencies.scala, jenkins builds, etc... already used 2.12.14 and 2.13.6)

Reply:
https://github.com/akka/akka/pull/30456